tbmalt/structures/geometry.py

```diff
@@ -33,6 +34,8 @@ class Geometry:
         cells: Lattice vectors of the periodic systems. [DEFAULT: None]
         frac: Whether using fractional coordinates to describe periodic
             systems. [DEFAULT: False]
+        cutoff: Global cutoff for the diatomic interactions in periodic
```

Comment:
Hi Adam, I have a look of this cutoff argument. It is used when generating the periodic images, and the value of this cutoff depends on the number of grid points of the sk table used. Since the Periodic object is now attached to the Geometry and the Geometry object is kind of independent with SkFeed if I understand correctly, I guess it will be hard to pass the information of the sk table to Geometry directly. In this case, maybe manually inputting a cutoff distance for Geometry can be a compromise way. In DFTB+, the cutoff will be determined during the process of initializing program, which reads the count of grid points of the sk parameter used. Maybe I am wrong or if you have any ideas, please let me know. Thank you.